src/ansys/fluent/core/session.py

```diff
@@ -124,7 +124,7 @@ def build_from_fluent_connection(
         self._fluent_connection = fluent_connection
         self._file_transfer_service = file_transfer_service
         self._error_state = fluent_connection._error_state
-        self.scheme_eval = fluent_connection.scheme_eval
+        self.scheme_eval = fluent_connection._connection_interface.scheme_eval
```

Comment:
`scheme_eval` can be an independent construction argument.

Reply:
@seanpearsonuk, should this be done now or during next level of refactoring when we touch session. The plan was to keep this PR specific to FluentConnection.

Reply:
I will leave it up to you.

Reply:
Sure, I'll look into it.